apps/api-example/src/app/slack-mail-provider.ts

```diff
@@ -20,8 +14,8 @@ export class SlackMailProvider extends BaseMailProvider {
     this.webhookURL = props.webhookURL
   }
 
-  async webhookForSendMail({req}: WebhookForSendMailProps): Promise<MailLogStatus[]> {
-    return Promise.resolve([])
+  async webhookForSendMail() {
+    return []
```

Comment:
should it stay like this?

Reply:
Yeah, slack doesn't give webhooks for sent messages back